numpy/core/tests/test_umath.py

```diff
@@ -1861,8 +1862,8 @@ class A(object):
             def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
                 return self, ufunc, method, inputs, kwargs
 
+        inner1d = ncu_tests.inner1d
```

Comment:
Is there a good reason not to prefer the long, but more informative, `inner1d = np.core.umath_tests.inner1d`?